File: dart/1010.dart
import 'dart:io';

void main() {
  List<String> values = stdin.readLineSync().split(' ');
  int code = int.parse(values[0]);
  int unit = int.parse(values[1]);
  double price1 = double.parse(values[2]);

  values = stdin.readLineSync().split(' ');
  code = int.parse(values[0]);
  int unit2 = int.parse(values[1]);
  double price2 = double.parse(values[2]);

  double valor = unit * price1 + unit2 * price2;

  print("VALOR A PAGAR: R\$ ${valor.toStringAsFixed(2)}");
}

File: python/1010.py
values = input().split(' ')
code = int(values[0])
unit = int(values[1])
price1 = float(values[2])

values = input().split(' ')
code = int(values[0])
unit2 = int(values[1])
price2 = float(values[2])

valor = (unit * price1 + unit2 * price2)

print("VALOR A PAGAR: R$ %0.2f" % valor)